winpr/libwinpr/synch/wait.c

```diff
@@ -109,7 +109,7 @@ static long long ts_difftime(const struct timespec *o,
 #if !defined(HAVE_PTHREAD_GNU_EXT)
 #include <pthread.h>
 
-#if defined(__FreeBSD__) || defined(sun)
+#if defined(__FreeBSD__) || defined(sun) || defined(__OpenBSD__)
```

Comment:
OK.  I can make that change and re-test.  How would you recommend I do
this?  Should we let this one go through I have me create a separate pull
request?  Or do you want to reject this one and have me add this in and
re-submit the pull request?


Thanks,
Bryan

On Tue, Apr 21, 2015 at 3:46 PM, akallabeth <notifications@github.com>
wrote:

> In winpr/libwinpr/synch/wait.c
> <https://github.com/FreeRDP/FreeRDP/pull/2562#discussion_r28814906>:
>
> > @@ -109,7 +109,7 @@ static long long ts_difftime(const struct timespec *o,
> >  #if !defined(HAVE_PTHREAD_GNU_EXT)
> >  #include <pthread.h>
> >
> > -#if defined(__FreeBSD__) || defined(sun)
> > +#if defined(__FreeBSD__) || defined(sun) || defined(__OpenBSD__)
>
> So there is actually an implementation available on OpenBSD? Then this
> custom implementation should not be used at all, it is meant for systems
> lacking one.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/FreeRDP/FreeRDP/pull/2562/files#r28814906>.
>